### PR TITLE
[Streaming]Renaming streaming context

### PR DESCRIPTION
**Pull request by JingGe**
## Why are these changes needed?

According to the naming convention found in the internet, e.g. wikipedia, ververica(https://www.ververica.com/what-is-stream-processing), It makes sense to rename the StreamingContext to StreamContext. Relevant discussion please refer to: PR: https://github.com/ray-project/ray/pull/7512

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Comment by JingGe** [comment]
@chaokunyang @TisonKun

**Review by tisonkun — APPROVED** [review]
Ref discussion https://github.com/ray-project/ray/pull/7512#discussion_r391444155

LGTM +1

**Comment by JingGe** [comment]
@raulchen Could you help merging it？Thanks.

**Comment by chaokunyang** [comment]
I think `streaming` here represents all streams in a streaming application, so should we rename `StreamingContext` to `StreamContext`. I think `StreamingContext` is appropriate as entry point of streaming application. It's called same name in `spark` too. And in `kafka-streams`, it's called `StreamsXXX`.

**Comment by chaokunyang** [comment]
@MissiontoMars @BalaBalaYi @yangjianzhang @ ashione  what's your suggestion?

File: streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/python/PythonGateway.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public PythonGateway() {
 
   public byte[] createStreamingContext() {
     streamingContext = StreamingContext.buildContext();
```
Comment (zhongchun) [inline streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/python/PythonGateway.java]: I think `streamingContext = StreamingContext.buildContext();` should be `streamContext = StreamContext.buildContext();`

Comment (JingGe) [reply streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/python/PythonGateway.java]: Nice catch, Thanks! 
It looks like the IDE refactoring didn't work properly. Fixed with many other classes together.

**Pull request closed without merging**